npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater_spec.rb

```diff
@@ -1089,7 +1089,7 @@
     let(:previous_requirements) { requirements }
 
     it "raises a helpful error" do
-      expect { updated_npm_lock_content }.to raise_error(Dependabot::DependencyFileNotResolvable)
+      expect { updated_npm_lock_content }.to raise_error(Dependabot::InconsistentRegistryResponse)
     end
```

Comment:
The error being thrown has changed because the `lockfileVersion` for these dependency files is `3`. Previously, the spec used npm `8`, but with this update, it now defaults to npm `10` to align with the latest version for `lockfileVersion` 3 and above.  

Example lockfile:  

```json
{
  "name": "DependencyNotFound",
  "version": "0.36.1",
  "lockfileVersion": 3,
  "requires": true,
  "packages": {
    "": {
      "name": "DependencyNotFound",
      "version": "0.36.1",
      "license": "MIT",
      "dependencies": {},
      "devDependencies": {
        "eslint": "^9.5.0"
      },
      "engines": {
        "node": ">=12"
      }
    }
  }
}
```